### ISSUE TITLE
spread loss is not decreasing

### ISSUE THREAD

**Issue by dragonfly90** [www0wwwjs1/Matrix-Capsules-EM-Tensorflow#20]
I tried python3 train.py "mnist" with master version and the default parameters. But the spread loss is not decreasing, it jumps from 0.36 to 0.65 in 1200 iterations. However, the experiment version seems to work fine.

**Comment by convnets** [comment]
@dragonfly90  Have you figured out why spread loss is increasing ?  I have encountered the same issue.

**Comment by dragonfly90** [comment]
@convnets  I tried config.iter_routing=2, it works. But I tried config.iter_routing=3, it does not work.

**Comment by ashleylid** [comment]
I have a question about the spread loss too - it seems to be directly related to the all_loss. I am trying to train on my own image input data, it trains for a while, the loss decreases to like 0.17 and then after say 1000 iterations it jumps up to 0.3 and then starts decreasing again. I have 1570 training images of 5 classes with one instance each in the same format at smallNORB  - let me know if I should take this over to a new thread..
